lib/src/pubspec.dart

```diff
@@ -388,6 +388,43 @@ class Pubspec {
   bool _parsedPublishTo = false;
   String _publishTo;
 
+  /// The list of patterns covering _false-positive secrets_ in the package.
+  ///
+  /// This is a list of git-ignore style patterns for files that should be
+  /// ignored when trying to detect possible leaks of secrets during
+  /// package publication.
+  List<String> get falseSecrets {
```

Comment:
Should we consider having this as a separate file (or even just using .gitallowed?) to not get too many keys into the pubspec?

Reply:
I still think this is the wrong place for these patterns.
They don't really describe the package in a way that is relevant to most users of a pubspec.

Reply:
Summary of debate from chat regarding putting it in another file:
 * Publishing dot-files is something we generally avoid.
   * But we want the list of false-positives to be published, so that we can see that something was a false positive secret.
 * Many top-level configuration files is undesirable
 * The file would need a long name that includes `pub` otherwise, you can't know what it belongs to.

Reply:
Downside:
  * Can't place a `pub-false-secrets` file in a sub-folder
  * Writing git ignore style patterns in a YAML list is a bit weird.
     * (People will hopefully use absolute path most of the time)